crates/fuzzing/src/generators.rs

```diff
@@ -52,6 +52,7 @@ impl Arbitrary for WasmOptTtf {
 pub struct DifferentialConfig {
     strategy: DifferentialStrategy,
     opt_level: DifferentialOptLevel,
+    multi_value: bool,
```

Comment:
I don't think it makes sense to do differential testing between a config with and without multi-value enabled. At best, one side will instantiate and run the wasm OK and the other won't (if the wasm uses multi-value) or they will both instantiate and run the wasm OK (if the wasm doesn't use multi-value). We're not really testing and comparing different compiler pipelines (eg with vs without optimizations) in this scenario, which is what differential testing is supposed to exercise.